fs-storage/src/file_storage.rs

```diff
@@ -299,34 +355,35 @@ mod tests {
         let storage_path = temp_dir.path().join("teststorage.txt");
 
         let mut file_storage =
-            FileStorage::new("TestStorage".to_string(), &storage_path);
+            FileStorage::new("TestStorage".to_string(), &storage_path).unwrap();
         file_storage.write_fs().unwrap();
-        assert_eq!(file_storage.needs_syncing().unwrap(), false);
+        assert_eq!(file_storage.needs_syncing().unwrap(), SyncStatus::NoSync);
         std::thread::sleep(std::time::Duration::from_secs(1));
         file_storage.set("key1".to_string(), "value1".to_string());
-        assert_eq!(file_storage.needs_syncing().unwrap(), true);
+        assert_eq!(file_storage.needs_syncing().unwrap(), SyncStatus::DownSync);
```

Comment:
Thank you for adding the tests. 

Could you include additional tests for the `SyncStatus::UpSync` and `SyncStatus::FullSync` cases?